src/string_utils.cpp

```diff
@@ -4,6 +4,8 @@
 #include <cctype>    // for tolower, isspace
 #include <sstream>
 
+#include "xtensor/xarray.hpp"
```

Comment:
Now that we're using a template for `concatenate` . We shouldn't need to include this header anymore.